### DISCUSSION TITLE
One Window In NZP!!!!!!

### DISCUSSION THREAD

**Discussion by Galvinh0slk4** [nzp-team/nzportable#997] [Map Releases]
Here's a map I created (I'm new to this) using Trenchbroom, a map where you have only one window, with a wall gun and 8 perks, PAP, and Mystery Box, test your limits on that map.
[OneWindow.zip](https://www.mediafire.com/file/dco2c7xaiop4efl/[OneWindow](https://www.mediafire.com/file/dco2c7xaiop4efl/OneWindow.zip/file).zip/file)
![OneWindow](https://github.com/user-attachments/assets/9e4f4265-f446-4064-a9b9-5c455204ca9f)

**Comment by mmkkiinn** [comment]
Please create a psp version its a cool map 😭😭🙏🏻🙏🏻🙏🏻

**Reply by MotoLegacy** [reply]
Remember what I told you about filing game issues for map compatibility, instead of making these sorts of requests in the map thread?

**Reply by mmkkiinn** [reply]
But its a little problem

**Comment by BrownNPC** [comment]
MICHAEL ZOMBIES MENTIONED

**Reply by BrownNPC** [reply]
Yo can you try to make the lighthouse map from Michael zombies too?

**Comment by Lolinburgs** [comment]
Really fun map but you can easily run into glitch spots so I compiled some I found: 

https://github.com/user-attachments/assets/f905b457-f846-4e79-bfbb-7f8563fb4055
- This is mainly an issue with the fact that zombies can land on your head. So it's mainly the issue of the devs ig but I'd probably not have the window up there. I also like rebuilding the window and you can't cause its up there so.


https://github.com/user-attachments/assets/adb7599c-51d0-4af6-92a5-8a46af576dd9
- I have no idea if you can fix this or not it may be an issue with just the game itself.


https://github.com/user-attachments/assets/ab851f63-be90-4564-8389-f7a3ff87575d
- Not being able to go behind the perk machines would fix this but also the dog ai should also be made not to be able to fall off the map so that part of it isn't your fault obviously 

https://github.com/user-attachments/assets/0068b5ec-0b86-4fac-a691-80f1bdc6cab8
- Again fixing going behind the perk machines would be nice but also the zombie ai shouldn't just kill themselves lol (again as a map creator I don't think you can do anything about that?)

https://github.com/user-attachments/assets/7d0f97f9-a8be-40b4-b5c0-d17cf9d5a1bb
- they seem to pile up on the top if you get to a high enough round. You could probably fix this if it was a bigger area. again issue in ai too though.

**Comment by spectrum1-wave** [comment]
is the link down? I cant play the map.

**Comment by irandiaz** [comment]
If you can, would you make a PSP version or is there already one?

**Reply by MotoLegacy** [reply]
If a map doesn't work on a given platform you should probably file a bug report for it to be investigated, not pester the map maker

**Reply by irandiaz** [reply]
Ok Thanks for the heads up, I didn't mean to be annoying, but it won't happen again.

**Comment by irandiaz** [comment]
This map is great, it's very entertaining.

**Comment by ProfessorZIRE** [comment]
make the barrier repairable and make it so i dont die when i fall
